components/post_view/message_attachments/message_attachment/message_attachment.jsx

```diff
@@ -315,6 +316,7 @@ export default class MessageAttachment extends React.PureComponent {
                             options={{
                                 mentionHighlight: false,
                                 markdown: false,
```

Comment:
Do we still need the `markdown: false` here? Since you're specifying the renderer, I don't think this line does anything

Reply:
This is indeed unnecessary. I removed it int the last update.